src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+package main.java;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+                + "|  _ \\ _   _| | _____\n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Hello! I'm Duke\n     What can I do for you?");
```

Comment:
different parts of the code can be grouped together as a method, make it easier for abstraction.